src/components/Layout/FormPageLayout.tsx

```diff
@@ -85,11 +86,12 @@ export const FormPageLayout = React.memo(FormPageLayoutComponent) as typeof Form
 
 function PageHeader<F>(props: FormPageLayoutProps<F>) {
   const { pageTitle, breadCrumb, submitAction, cancelAction, tertiaryAction, formState } = props;
-
+  const { notice } = useToastContext();
   const tids = useTestIds(props);
 
   return (
-    <header css={Css.gr(1).gc("1 / 4").sticky.top0.hPx(headerHeightPx).bgWhite.z5.$} {...tids}>
```

Comment:
@blambillotte removing`hPx` allows to effectively "push down" components below the header when the `BannerNotice` is rendered, but it also disrupts the scroll behavior of the left nav section:

https://github.com/user-attachments/assets/1dfaad8a-d656-453b-84c7-998148a00537

We could probably make it work even with the banner rendered, but that would require making `headerHeightPx` dynamic and maybe pass down a ref to `Banner` to recalc the current height of that component. But I think would overcomplicate things, since it's only a temporary state (and IMO the banner is more disruptive to the UI than the scroll being partially hidden).

I'd like to know your thoughts though.

Reply:
hmm yeah tricky! I tend to agree that needing to use javascript to do DOM sizing, etc is something to avoid if we can. This seems like a fine solution though happy to look at it with you to consider options - I did notice the right panel isn't growing to fill the full viewport height like it was before so there was some other style regression at some point on this layout